kornia/geometry/conversions.py

```diff
@@ -1478,3 +1479,35 @@ def ARKitQTVecs_to_ColmapQTVecs(qvec: Tensor, tvec: Tensor) -> tuple[Tensor, Ten
     t_colmap = t_colmap.reshape(-1, 3, 1)
     q_colmap = rotation_matrix_to_quaternion(R_colmap.contiguous())
     return q_colmap, t_colmap
+
+
+def vector_to_skew_symmetric_matrix(vec: Tensor) -> Tensor:
+    r"""Converts a vector to a skew symmetric matrix.
```

Comment:
can you add some formulas ? https://math.stackexchange.com/questions/258775/from-a-vector-to-a-skew-symmetric-matrix